python/ray/job_config.py

```diff
@@ -17,6 +17,9 @@ class JobConfig:
         runtime_env (dict): A runtime environment dictionary (see
             ``runtime_env.py`` for detailed documentation).
         client_job (bool): A boolean represent the source of the job.
+        max_pending_calls (int): The max actor calls pending in the sender's
+            client queue when back pressure occurs. -1 means the pending count
+            is unlimited and back pressure disabled.
```

Comment:
Can you please make this a per-actor configuration rather than putting it in the JobConfig?

Reply:
Both job config and per-actor config are supported, should we remove the job config ?

Reply:
I think we should remove the job_config, it doesn't match other actor-level configs we have like max_retries. Can always add it later!